cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -122,12 +122,43 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
-    {# Google Tag Manager #}
+
+    <!-- Google Optimize page-hiding snippet -->
+    <style>
```

Comment:
I'm unclear if your question is specific to the page hiding snippet, or the concept of A/B testing in general. What do you mean by "outside of a blog post"?